ebodac/src/test/java/org/motechproject/ebodac/uitest/test/DisplayOfSubjectVisitInTabularFormatUiTest.java

```diff
@@ -0,0 +1,48 @@
+package org.motechproject.ebodac.uitest.test;
+
+import org.apache.commons.lang.StringUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.motech.page.LoginPage;
+import org.motech.test.TestBase;
+import org.motechproject.ebodac.uitest.page.HomePage;
+import org.motechproject.ebodac.uitest.page.ParticipantEditPage;
+import org.motechproject.ebodac.uitest.page.ParticipantPage;
+
+import static org.junit.Assert.assertTrue;
+
+
+public class DisplayOfSubjectVisitInTabularFormatUiTest extends TestBase {
+    private LoginPage loginPage;
+    private HomePage homePage;
+    private ParticipantPage participantPage;
+    private ParticipantEditPage participantEditPage;
+    private String user;
+    private String password;
+
+    @Before
+    public void setUp() {
+        user = properties.getUserName();
+        password = properties.getPassword();
+        loginPage = new LoginPage(driver);
+        homePage = new HomePage(driver);
+        participantPage = new ParticipantPage(driver);
+        participantEditPage = new ParticipantEditPage(driver);
+        if (StringUtils.equals(homePage.expectedUrlPath(), currentPage().urlPath())) {
```

Comment:
it was "!=" before and now you have equals, should not this be !StringUtils.equals?